### DISCUSSION TITLE
gulp watch / build doesnt generate css from scss first time - I have to deliberately adjust scss and change it back to get gulp to see change and generate css

### DISCUSSION THREAD

**Discussion by therobyouknow** [gulpjs/gulp#2557] [Help]
how do I get gulp watch to generate css from scss without having to "artificially" tweak the scss - i.e. change it to only change it back for a change to be detected to trigger gulp watch / build to generate css

**Comment by thecodercoder** [comment]
For the first time, make sure you are running your SCSS task as part of your Gulp default task. That way, when you run `gulp` on the command line to begin Gulp, it will compile SCSS to CSS. Then your watch task will watch and compile any future SCSS changes.

Here's a simplified version of what I'm using for my own default Gulp task:

`exports.default = series(scssTask, jsTask, watchTask);`

It will run my SCSS and JS task and also start the watch task. The watch task will then also run the SCSS and JS tasks when any changes are detected. Hope this helps.
